File: src/test/scala/com/martomate/tripaint/InjectiveHashMapTest.scala
package com.martomate.tripaint

import com.martomate.tripaint.util.{InjectiveHashMap, InjectiveMap}

class InjectiveHashMapTest extends InjectiveMapTest {
  override def createMap[L, R]: InjectiveMap[L, R] = new InjectiveHashMap[L, R]
}

File: core/src/test/kotlin/tripaint/InjectiveHashMapTest.kt
package tripaint

import tripaint.util.InjectiveHashMap
import tripaint.util.InjectiveMap
import tripaint.util.InjectiveMapTest

class InjectiveHashMapTest : InjectiveMapTest() {
    override fun <L, R> createMap(): InjectiveMap<L, R> = InjectiveHashMap<L, R>()
}
